content/sensu-core/2.0/getting-started/installation-and-configuration.md

```diff
@@ -94,20 +115,51 @@ sudo /etc/init.d/sensu-backend start
 sudo /etc/init.d/sensu-agent start
 {{< /highlight >}}
 
-#### Ubuntu 16.04 / CentOS 7 / RHEL 7
+### Ubuntu 16.04 / CentOS 7 / RHEL 7
 
 Start the services using systemd.
 
 {{< highlight shell >}}
 sudo systemctl start sensu-backend
 sudo systemctl start sensu-agent
 {{< /highlight >}}
+{{< platformBlockClose >}}
 
+{{< platformBlock "Windows" >}}
 ### Windows
 
 Coming soon.
+{{< platformBlockClose >}}
 
-### Docker
+## Validating the services
+
+{{< platformBlock "Ubuntu/Debian RHEL/CentOS" >}}
+### Ubuntu 14.04 / CentOS 6 / RHEL 6
+
+Verify that the services are properly running using `journalctl`.
+
+{{< highlight shell >}}
+journalctl -u sensu-backend -f
+journalctl -u sensu-agent -f
+{{< /highlight >}}
+
+### Ubuntu 16.04 / CentOS 7 / RHEL 7
+
+Verify that the services are properly running using the log files.
+
+{{< highlight shell >}}
+tail -f /var/log/sensu/sensu-backend.log
```

Comment:
We'll need to use journalctl for accessing logs in 16.04 as well.

Reply:
Oh good catch, I'm realizing I have inverted these two sections!